Update terra-avatar (#1273)
cerner/terra-framework: packages/terra-application-navigation/src/terra-dev-site/doc/application-navigation/About.1.doc.mdx

The `userConfig` prop allows consumers to set a user associated to the current application context. The ApplicationNavigation will render this content in different ways based on the active responsive breakpoint.

<Notice >

> If neither initials or the imageSrc are  provided, the avatar will fall back to the generic user avatar. If initials not provided, imageSrc is provided, but fails the avatar will fall back to the first two letters of the name. Initials will be required with the next major version of application navigation, provided via terra-application v2.

Author: ```suggestion
If neither initials or the imageSrc are  provided, the avatar will fall back to the generic user avatar. If initials are not provided and imageSrc is provided, but fails, the avatar will fall back to the first two letters of the provided name. Initials will be required with the next major version of application navigation, provided via terra-application v2.
```

> If neither initials or the imageSrc are  provided, the avatar will fall back to the generic user avatar. If initials are not provided and imageSrc is provided, but fails, the avatar will fall back to the first two letters of the provided name. Initials will be required with the next major version of application navigation, provided via terra-application v2.

Reviewer: ```suggestion
It is recommended that `initials` prop be provided for all cases if `userConfig` is used. It will be updated to **Required** with the next major version of application navigation, provided via terra-application v2.

*Note: If both `initials` and `imageSrc` are not provided, the avatar will fall back to the generic user avatar. If `initials` are not provided, but `imageSrc` is provided and fails, the avatar will fall back to use first two letters of the provided `name` (not preferred) in place of showing the `initials` (preferred).* 
```

The `userConfig` prop allows consumers to set a user associated to the current application context. The ApplicationNavigation will render this content in different ways based on the active responsive breakpoint.

> <Notice >

Author: ```suggestion
<Notice ariaLevel="5">
```
